BossExpertiseSystem.cs

```diff
@@ -6,36 +6,44 @@ namespace BossExpertise
 {
     class BossExpertiseSystem : ModSystem
 	{
-        public override void OnWorldLoad()
+		static public Difficulty worldDifficulty = new();
```

Comment:
I propose renaming this to `OriginalDifficulty`.

Reply:
Ok, but I will have to modify that when I allow the command to modify the world. So that the before and afters can correctly set the new level.

Reply:
Mm yeah true. On second thought, WorldDifficulty is fine.